manifests/init.pp

```diff
@@ -199,7 +199,12 @@
   }
 
   # cleanup old kernels
-  ensure_packages(['yum-utils'])
+  if (($facts['os']['name'] == 'RedHat' or $facts['os']['name'] == 'CentOS') and $facts['os']['release']['major'] == '8') {
```

Comment:
`if $facts['os']['name'] in ['RedHat', 'CentOS']` should work as well.

do we even need to test on the os name? Isn't the release number enough?